recipe/conda_build_config.yaml

```diff
@@ -1,4 +1,3 @@
 
 node:
-  - 8
-  - 6
+  - 11
```

Comment:
Oops! Didn't catch that this dropped the node matrix. Will fix that in a later PR.